apps/prosody/templates/services/nginx/configs/prosody.conf.j2

```diff
@@ -37,7 +37,8 @@ server {
     proxy_set_header Host $host;
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
-    proxy_read_timeout 86400;
+    proxy_read_timeout {{ prosody_nginx_proxy_read_tiemout }};
+    proxy_send_timeout {{ prosody_nginx_proxy_send_tiemout }};
```

Comment:
```suggestion
    proxy_read_timeout {{ prosody_nginx_proxy_read_timeout }};
    proxy_send_timeout {{ prosody_nginx_proxy_send_timeout }};
```

(+ other occurences in ingress.yml.j2 and main.yml)

Reply:
It's already fixed, I saw the typo when I opened the PR... shame on me.